scripts/update_dep.sh

```diff
@@ -26,21 +26,64 @@ set -euo pipefail
 
 source ./scripts/test_lib.sh
 
+if [ "$#" -lt 1 ] || [ "$#" -gt 2 ]; then
+    log_error "Illegal number of parameters. Usage: $0 module [version]"
+    exit 1
+fi
+
 mod="$1"
-ver="$2"
+ver="${2:-}"
+
+function print_current_dep_version {
+  log_info "${mod} version in all go.mod files:"
+  find . -name go.mod -exec grep -H "^\s*${mod}\s" {} + | sed 's|:|\t|' || true
+  printf "\n"
+}
+
+function is_fully_indirect {
+  local result
+  result=$(find . -name go.mod -print0 | xargs -0 -I{} /bin/sh -c "cd \$(dirname {}); go list -f \"{{if eq .Path \\\"${mod}\\\"}}{{.Indirect}}{{end}}\" -m all" | sort | uniq)
+  if [ "$result" = "true" ] ; then
+    return 0
+  else
+    return 1
+  fi
+}
```

Comment:
I think this can be simplified by returning the result from find, and later, instead of checking just with the value, compare with `= "true"`.